olm-catalog/serverless-operator/project.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     # is ignored as it is overridden by fake version via KUBERNETES_MIN_VERSION.
     # This value is used for CSV's min version validation.
     minVersion: 1.21.0
-  golang: '1.18'
+  golang: '1.19'
```

Comment:
https://github.com/openshift-knative/serverless-operator/pull/2032 didn't fix the issues?
cc @ReToCode

Reply:
No, the issue was only on 1.19, but the `rm -rf` below will fix it.